tg/CoreData/CoreDataManager.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  CoreDataManager.swift
+//  tg
+//
+//  Created by Apple on 6.02.22.
+//
+
+import Foundation
+import UIKit
+import CoreData
+
+class CoreDataManager {
+    static let dataManager: CoreDataManager = CoreDataManager()
```

Comment:
Please read about Singleton pattern. If you use it class has to have private `init()`